pkg/util/mount/mount_windows.go

```diff
@@ -204,3 +242,29 @@ func getDriveLetterByDiskNumber(diskNum string, exec Exec) (string, error) {
 	}
 	return string(output)[:1], nil
 }
+
+// getAllParentLinks walks all symbolic links and return all the parent targets recursively
+func getAllParentLinks(path string) ([]string, error) {
+	links := []string{}
+	for {
+		links = append(links, path)
+		if len(links) > 255 {
```

Comment:
Windows used to have a limitation in the headers of MAX_PATH, but that only applies to some older APIs now. This limit check shouldn't be needed as long as the right APIs are used

Reply:
In current scenario, the length would be only 3, set as 255 to avoid recursively link, e.g. a-->b, b-->c, c-->a

Reply:
can you make a constant for 255 or at least put some comments related to this number.

Reply:
is 255 a random picked number?

Reply:
@jingxu97 thanks for the review, 255 is actually a limit of symlink length in windows, see 
https://github.com/moby/moby/blob/master/pkg/symlink/fs_windows.go#L83
I have added a const for 255, pls check again, thx